src/sentry/api/endpoints/project_platforms.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+from rest_framework import serializers
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import ProjectPlatform
+
+
+class ProjectPlatformSerializer(serializers.Serializer):
+    platform = serializers.CharField()
+
+
+class ProjectPlatformsEndpoint(ProjectEndpoint):
+    def get(self, request, project):
```

Comment:
i would say that if we're going to keep this, change the format of the response to be:

```json
[{
    "platform": "php",
    "dateCreated": "date_added",
}]
```